cmd/server/app/options/options.go

```diff
@@ -41,6 +41,9 @@ type ProxyRunOptions struct {
 	// pings the client to see if the transport is still alive.
 	KeepaliveTime         time.Duration
 	FrontendKeepaliveTime time.Duration
+	// Time that a gRPC connection can be inactive (no pkt received) before
```

Comment:
"gRPC connection" is a bit confusing. We have a long lived gRPC connection between the KonnServer and KonnAgent which makes up part of the tunnel. I think it would be good to elaborate on the connections we are effecting. Proxied connections? Request connections? Client connections?

Reply:
Thanks for this feedback. This is about the idle time between frontend and the proxy-server in which no packet type is transferred on the stream.

Initially I wanted to reuse the FrontendKeepaliveTime, but I felt making it separate would be much better to make this approach opt-in.